### ISSUE TITLE
Microsoft store undertale use

### ISSUE THREAD

**Issue by IndependantFlower** [RedSpah/HATE#16]
I really want to use this, but I bought undertale from the microsoft stor, and I don't know how to download it from there. I couldn't find any tutorials online, and I would really appreciate the help.

Thank you!

**Comment by redspah** [comment]
Try this?

https://support.microsoft.com/en-us/account-billing/i-can-t-find-or-install-an-app-from-microsoft-store-ca530de4-cd19-f728-dae7-7368fab8df4d

**Comment by IndependantFlower** [comment]
No, I meant that I downloaded undertale, but I don't know how to use hate.exe with this version of it
